src/google_calendar_tool.js

```diff
@@ -48,44 +36,62 @@ export class GoogleCalendarTool extends Tool {
 
     return auth
   }
+}
 
-  async runClassification(query) {
-    const prompt = new PromptTemplate({
-      template: EVENT_CLASSIFICATION_PROMPT,
-      inputVariables: ['query']
+export class GoogleCalendarCreateTool extends GoogleCalendarBase {
+  constructor(params) {
+    super(params)
+    Object.defineProperty(this, 'name', {
+      enumerable: true,
+      configurable: true,
+      writable: true,
+      value: 'google_calendar_create'
     })
-    const createEventChain = new LLMChain({
-      llm: this.getModel(),
-      prompt,
-      verbose: true
+    Object.defineProperty(this, 'description', {
+      enumerable: true,
+      configurable: true,
+      writable: true,
+      value: CREATE_TOOL_DESCRIPTION
     })
-
-    return (await createEventChain.call({ query })).text
   }
 
   async _call(query) {
     const auth = await this.getAuth()
     const model = this.getModel()
 
-    const classification = await this.runClassification(query)
+    return await runCreateEvent(query, {
+      auth,
+      model,
+      calendarId: this.CALENDAR_ID
+    })
+  }
+}
 
-    console.log('Detected classification: ', classification)
+export class GoogleCalendarViewTool extends GoogleCalendarBase {
+  constructor(params) {
+    super(params)
+    Object.defineProperty(this, 'name', {
+      enumerable: true,
+      configurable: true,
+      writable: true,
+      value: 'google_calendar_view'
+    })
+    Object.defineProperty(this, 'description', {
+      enumerable: true,
+      configurable: true,
+      writable: true,
+      value: VIEW_TOOL_DESCRIPTION
+    })
+  }
```

Comment:
If all the properties (enumerable, configurable, writable) are `true`, it can be indeed replaced with the simple notation.

For now I was following the style of defining properties that was used in other tools. I will update it.